v1.14/alpine/fluent.conf

```diff
@@ -0,0 +1,23 @@
+<source>
```

Comment:
I'd like to see this linted as well as a section in the README.md explaining the default configuration and how to extend it. I'm not a fluentd expert so not sure if you can easily just mount extra files to use into the container rather than having to overwrite the whole config?

Reply:
We could, but this is 100% meant to be overridden by all users and is only there to make the image runnable.